code/game/objects/items/holy_weapons.dm

```diff
@@ -375,6 +375,7 @@
 	attack_verb = list("sawed", "tore", "cut", "chopped", "diced")
 	hitsound = 'sound/weapons/chainsawhit.ogg'
 	tool_behaviour = TOOL_SAW
+	block_upgrade_walk = 0
 	toolspeed = 1.5 //slower than a real saw
```

Comment:
Why nerf only the chainsaw sword when every other claymore nullrod has exactly the same stats? (Claymore, darkblade, force weapon, energy sword, etc.)

Reply:
Ah, i'd meant to nerf the chainsaw arm instead